gnd/src/main/java/com/google/android/gnd/repository/PolygonDialogInfoRepository.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.repository;
+
+import com.google.android.gnd.persistence.local.LocalValueStore;
+import javax.inject.Inject;
+
+public class PolygonDialogInfoRepository {
```

Comment:
Not sure we need a new repository just for this use case - maybe FeatureRepository would be the right place for this?

Reply:
removed and updated FeatureRepository.